src/Kunstmaan/AdminBundle/Entity/Exception.php

```diff
@@ -0,0 +1,226 @@
+<?php
+
+namespace Kunstmaan\AdminBundle\Entity;
+
+use DateTimeInterface;
+use Doctrine\ORM\Mapping as ORM;
+use Symfony\Bridge\Doctrine\Validator\Constraints\UniqueEntity;
+
+/**
+ * @ORM\Entity(repositoryClass="Kunstmaan\AdminBundle\Repository\ExceptionRepository")
+ * @ORM\Table(
+ *      name="kuma_exception",
+ *      indexes={
+ *          @ORM\Index(name="idx_exception_is_mark", columns={"is_mark"})
+ *      }
+ * )
+ * @UniqueEntity("hash")
+ * @ORM\HasLifecycleCallbacks()
+ */
+class Exception extends AbstractEntity
+{
+    /**
+     * @var string
+     *
+     * @ORM\Column(type="string", length=3)
+     */
+    private $code;
+
+    /**
+     * @var string
+     *
+     * @ORM\Column(type="string")
+     */
+    private $url;
+
+    /**
+     * @var string
+     *
+     * @ORM\Column(type="string", nullable=true)
+     */
+    private $urlReferer;
+
+    /**
+     * @var string
+     *
+     * @ORM\Column(type="string", nullable=false, unique=true, length=32)
+     */
+    private $hash;
+
+    /**
+     * @var int
+     *
+     * @ORM\Column(type="integer", nullable=false)
+     */
+    private $events;
+
+    /**
+     * @var bool
+     *
+     * @ORM\Column(type="boolean", name="is_mark")
+     */
+    private $isMark;
```

Comment:
I went through the whole PR again and was thinking that it would make more sense to rename this param to ```resolved```. It creates a better context of what the meaning of the toggle is in the adminlist. If I say this exception is marked it's not really clear what it's marked for, but if I say that this exception is resolved, that is very clear for the administrator of the cms. Sorry for the hassle but this would be the cherry on top! I promise this is the last change I'm gonna ask for this PR :)

Keep up the good work, this is great stuff!